sample-code/src/orchestration.ts

```diff
@@ -208,3 +209,50 @@ export async function orchestrationRequestConfig(): Promise<OrchestrationRespons
     }
   );
 }
+
+/**
+ * Use the orchestration service with JSON obtained from AI Launchpad.
+ * @param filePath - Path to the JSON file obtained from AI Launchpad.
+ * @returns The orchestration service response.
+ */
+export async function orchestrationFromJSON(
+  filePath?: string
+): Promise<OrchestrationResponse> {
+  // Default JSON configuration
+  const defaultJsonConfig = `{
+    "module_configurations": {
+      "llm_module_config": {
+        "model_name": "gpt-35-turbo",
+        "model_params": {},
+        "model_version": "0613"
+      },
+      "templating_module_config": {
+        "template": [
+          {
+            "role": "user",
+            "content": "SAP"
+          },
+          {
+            "role": "system",
+            "content": "When given the name of a company, provide the year of its establishment."
+          }
+        ]
+      }
+    }
+  }`;
+
+  try {
+    // Use provided file path if available, else fallback to default configuration
+    const jsonConfig = filePath
+      ? await fs.promises.readFile(filePath, 'utf-8')
```

Comment:
[pp] I think I'd only implement the file variant, and then mention in a comment that providing the plain string in source code is also possible. Having a fallback logic makes the code a bit more complex and thus harder to understand. Compare the Java docs